### ISSUE TITLE
Sprint 8

### ISSUE THREAD

**Issue by lastmjs** [lastmjs/podcrypt#323]
- [x] Backing up the state to a json file stopped partway for me...
- [x] Add currently playing title, and potentially other useful information to the top bar
- [x] Handle the quota exceeded error gracefully...tell the users what to do
  - [x] navigator.storage.estimate().then((e) => console.log(e))
  - [x] Find the best API for determining storage
  - [x] Put something on the downloads page when there are no downloads
- [x] Make sure the marquee is working correctly on all screen sizes, I'm hoping the Chrome simulator thing was just wrong...make sure the text goes all the way out one side and all the way into the other
  - [x] The scrolling disappearance and reappearance is not consistent across mobile and desktop
  - [x] Do not freeze the main thread, the marquee lags
  - [x] Only scroll if the text is too big
- [x] Homepage
  - [x] Loading for each tab individually...try not to block if possible
  - [x] Instead of saying no results tell the user to search for podcasts above, just like the my podcasts tab
  - [x] Refactor pc-podcast-search-results completely...
  - [x] Add the tabindex to the global state so that it is persisted on refreshes
  - [x] It all seems to be working well, just make it smooth now
  - [x] When you hit search, change the tab index
  - [x] Consider dynamically adding the search results page...and consider not naming the tab search results, but naming it the search term (this can probably wait until 1.0 or 2.0)
  - [x] The homepage loads like 20MB of stuff!
  - [x] Add something to the top bar when an episode isn't playing
- [x] Downloading Wyre Talks episodes is broken. Only part of the episodes get downloaded, like 3 minutes worth
- [x] Backing up data download broke
  - [x] Programmatic clicks seem to be blocked on mobile browsers
- [x] Make a donation wallet web component...I already have need of this in two different projects...perhaps split it up into multiple components
- [x] Look into Netlify's built-in proxying...we might be able to do that instead of running out own proxy
